frame/nomination-pools/runtime-api/src/lib.rs

```diff
@@ -24,10 +24,19 @@ use codec::Codec;
 
 sp_api::decl_runtime_apis! {
 	/// Runtime api for accessing information about nomination pools.
+	#[api_version(1)]
```

Comment:
That is the default version.

Reply:
Hi Basti. This is the initial staking api, it does not exist prior to these PRs. So this is correct no?

Reply:
What Basti means is that if you don't put `#[api_version(1)]`, it is already there as default.